tests/testhelpers/generators.go

```diff
@@ -17,13 +17,15 @@ func NewRand() *rand.Rand {
 // Test Helpers that are useful for Generating random Scoot Api Structs
 // To help with testing the Scoot API and Scheduler
 
-// Generates a scoot.JobDefinition
-func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
-
+// Generates a scoot.JobDefinition with numTasks tasks (or random if numTasks == -1)
```

Comment:
Added to help string.

What bound should there be? Why bound on the client instead of on the server?

I was playing last night with 20k, which seems totally reasonable.